Networking/NetworkingTests/Remote/UserRemoteTests.swift

```diff
@@ -0,0 +1,40 @@
+import XCTest
+@testable import Networking
+
+
+/// UserRemote Unit Tests
+///
+final class UserRemoteTests: XCTestCase {
+
+    /// Dummy Network Wrapper
+    ///
+    let network = MockNetwork()
+
+    /// Repeat always!
+    ///
+    override func setUp() {
+        network.removeAllSimulatedResponses()
+    }
+
+    func test_loadUserInfo_correctly_returns_parsed_user() throws {
+        // Given
+        let remote = UserRemote(network: network)
+        let siteID: Int64 = 1234
+        let urlSuffix = "sites/\(siteID)/users/me"
+        network.simulateResponse(requestUrlSuffix: urlSuffix, filename: "user-complete")
+
+        // When
+        let result: Result<User, Error> = waitFor { promise in
+            remote.loadUserInfo(for: siteID) { result in
+                promise(result)
+            }
+        }
+
+        // Then
+        XCTAssertTrue(result.isSuccess)
+        let user = try XCTUnwrap(result.get())
+        XCTAssertEqual(user.siteID, siteID)
+    }
+
+
```

Comment:
Same as Network tests - can we add simple failure test case too?

Reply:
Will do! 👍

Reply:
Negative test added!